content/engine/install/sles.md

```diff
@@ -40,28 +40,23 @@ To get started with Docker Engine on SLES, make sure you
 To install Docker Engine, you need a maintained version of one of the following
 SLES versions:
 
-- SLES 15-SP3 on s390x (IBM Z)
 - SLES 15-SP4 on s390x (IBM Z)
+- SLES 15-SP5 on s390x (IBM Z)
 
-You must enable the [`SCC SUSE`](https://scc.suse.com/packages?name=SUSE%20Linux%20Enterprise%20Server&version=15.3&arch=s390x)
+You must enable the [`SCC SUSE`](https://scc.suse.com/packages?name=SUSE%20Linux%20Enterprise%20Server&version=15.5&arch=s390x)
 repositories.
 
-You must enable the [OpenSUSE `SELinux` repository](https://download.opensuse.org/repositories/security). This repository is not added by default, and you need to enable
-it for the version of SLES you are running. Run the following commands to add it:
+You must enable the [OpenSUSE `SELinux` repository](https://download.opensuse.org/repositories/security:/SELinux/). This repository is not added by default, and you need to enable it for the version of SLES you are running. Run the following commands to add it:
```

Comment:
This still mentions;

> and you need to enable it for the version of SLES you are running

As the instructions now seem to be version-independent; perhaps we can drop that part?

Reply:
sure, we'll update as per @dvdksn's suggestion.